packages/tasks_panel/src/view/task-tree-item.ts

```diff
@@ -1,27 +1,47 @@
 import { find } from "lodash";
-import { TreeItem, TreeItemCollapsibleState, Command, tasks } from "vscode";
+import { TreeItem, TreeItemCollapsibleState, Command, tasks, ThemeIcon } from "vscode";
+import { isMatchBuild, isMatchDeploy } from "../../src/utils/ws-folder";
 
 type TaskStatus = "idle" | "running";
 
-export class IntentTreeItem extends TreeItem {
-  constructor(public label: string, public collapsibleState: TreeItemCollapsibleState) {
+class BranchTreeItem extends TreeItem {
```

Comment:
Why is it called "BranchTreeItem"? what it represents?

Reply:
it represent a common branch in tasks tree. The `ProjectTreeItem` and `IntentTreeItem` extends it with its own properties.